skeleton_binding_script.py

```
@@ -0,0 +1,134 @@
+    # 2. Create the cylinder
+    cylinder_obj = create_cylinder()
+
+    # 3. Create the armature
+    armature_obj = create_armature()
+
+    # 4. Parent the cylinder to the armature with automatic weights
+    # Ensure cylinder is active and armature is selected
+    bpy.ops.object.select_all(action='DESELECT')
+    cylinder_obj.select_set(True)
+    armature_obj.select_set(True)
+    bpy.context.view_layer.objects.active = armature_obj # Armature should be the active object for parenting
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)